packages/sdk/test/common/relayer.ts

```diff
@@ -49,59 +49,50 @@ export async function waitForDepositAndFillV3_5({
     throw new Error("Unable to parse deposit logs");
   }
 
-  const { request } = await destinationPublicClient.simulateContract({
-    address: destinationSpokepoolAddress,
-    abi: spokePoolAbiV3_5,
-    functionName: "fillV3Relay",
-    args: [
-      {
-        ...deposit,
-        originChainId: BigInt(originPublicClient.chain.id),
-        depositId: Number(deposit.depositId),
-      },
-      BigInt(destinationPublicClient.chain.id),
-    ],
-    account: isAddressDefined(exclusiveRelayer)
-      ? exclusiveRelayer
-      : chainClient.account.address,
-  });
+  const relayerAddress = isAddressDefined(exclusiveRelayer)
+    ? exclusiveRelayer
+    : chainClient.account.address;
 
-  return await chainClient.writeContract(request);
-}
+  const v3RelayData = toV3RelayData(deposit);
 
-export async function waitForDepositAndFillV3({
-  depositReceipt,
-  acrossClient,
-  originPublicClient,
-  destinationPublicClient,
-  chainClient,
-}: RelayerParams) {
-  const destinationSpokepoolAddress = await acrossClient.getSpokePoolAddress(
-    destinationPublicClient.chain.id,
-  );
+  const args = [
+    v3RelayData,
+    BigInt(destinationPublicClient.chain.id),
+    addressToBytes32(relayerAddress),
+  ] as const;
 
-  const deposit = getDepositFromLogs({
-    originChainId: originPublicClient.chain.id,
-    receipt: depositReceipt,
-  });
+  // console.log(`Doing fill with args:`, args);
 
-  if (!deposit) {
-    throw new Error("Unable to parse deposit logs");
-  }
+  // const { request: approveRequest } = await simulateApproveTx({
+  //   walletClient: chainClient,
+  //   publicClient: destinationPublicClient,
+  //   spender: destinationSpokepoolAddress,
+  //   approvalAmount: maxUint256,
+  //   tokenAddress: v3RelayData.outputToken,
+  // });
+
+  // console.log("Simulating fill approval...");
+
+  // const approveTxHash = await chainClient.writeContract({
+  //   account: chainClient.account,
+  //   ...approveRequest,
+  // });
+
+  // console.log("Approving fill spend...");
+
+  // const fillApprovalReceipt =
+  //   await destinationPublicClient.waitForTransactionReceipt({
+  //     hash: approveTxHash,
+  //   });
+
+  // console.log("Fill Approval Success!", fillApprovalReceipt);
```

Comment:
Why do we comment this?

Reply:
Ah this needs to be reinstated actually.